### ISSUE TITLE
Finish Photon Bronson.

### ISSUE THREAD

**Issue by stuartshields** [Chassis/Photon#1]
Currently this only supports PHP5.

**Comment by BronsonQuick** [comment]
This never actually got finished so currently it supports zero versions of PHP

**Comment by BronsonQuick** [comment]
Hence the readme 😜

**Comment by rmccue** [comment]
😂

**Comment by stuartshields** [comment]
@BronsonQuick Half arse code... Again.

**Comment by BronsonQuick** [comment]
Haha the client that we might've needed this for pivoted part way through this so I stopped dev. If I needed to use it, it would be finished with 🐛s just for you 😍

**Comment by stuartshields** [comment]
I don't think we need it for the VIP extension. I'll let you know if it changes. Thanks brah!

**Comment by stuartshields** [comment]
@BronsonQuick Actually after working on another project and coming across old tickets I can say this is needed for the VIP extension.

**Comment by BronsonQuick** [comment]
@stuartshields Coolness. I figured you'd figure out you do need it eventually 🎉
